dev/com.ibm.jbatch.container/src/com/ibm/jbatch/container/ws/impl/WSJobRepositoryImpl.java

```diff
@@ -128,9 +128,14 @@ public WSJobExecution createJobExecution(long jobInstanceId, Properties jobParam
     @Override
     public List<WSJobInstance> getJobInstances(int page, int pageSize) {
 
```

Comment:
Is this needed now that the query helper path supports the new group security and the "boundary", no-additional-search params path?